modules/ansible/templates-ansible/configure-network-services/playbook-configure-network-services.yml.tftpl

```diff
@@ -13,7 +13,5 @@
     client_config: ${client_config}
 
   roles:
-  - role: ibm.power_linux_sap.powervs_client_enable_services
+  - role: ibm.power_linux_sap.configure_network_management_services
     when: client_config.keys()|length > 0
```

Comment:
add one more  condition. It should be or condition
server_config.keys()|length > 0